libs/execution_context/ebpf_link.c

```diff
@@ -133,7 +133,7 @@ ebpf_link_attach_program(ebpf_link_t* link, ebpf_program_t* program)
         goto Done;
     }
 
-    const ebpf_program_type_t* program_type = ebpf_program_type(program);
+    const ebpf_program_type_t* program_type = ebpf_program_type_uuid(program);
```

Comment:
It seems odd to have the function have uuid in the name but return a type that does not have uuid in the name.  They should be consistent.

Reply:
Changed name. Note that there is already existing functions that return ebpf_program_type_t and uses uuid in the function name. Changing either the existing type or function names would cause unnecessary churn in many source files and would clutter this PR. So going with the minimal change for now.